app/helpers/pageflow/entries_helper.rb

```diff
@@ -1,5 +1,12 @@
 module Pageflow
   module EntriesHelper
+    def pretty_entry_title(entry)
+      [].tap do |title|
+        title << entry.title
+        title << entry.theming.cname_domain if entry.theming.cname_domain.present?
+      end.join(' - ')
+    end
+
```

Comment:
This can be simplified to `[entry.title, entry.themging.cname_domain.presence].compact.join(' - ')`, right?